plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageDatabaseUserInterface.java

```diff
@@ -155,11 +190,23 @@ protected void performDefaults()
 
         notificationsEnabled.setSelection(store.getBoolean(ModelPreferences.NOTIFICATIONS_ENABLED));
         notificationsCloseDelay.setSelection(store.getInt(ModelPreferences.NOTIFICATIONS_CLOSE_DELAY_TIMEOUT));
+        final String string = store.getString(ModelPreferences.CLIENT_TIMEZONE);
+        if (string.isEmpty()) {
+            clientTimezone.setText(TimezoneRegistry.DEFAULT_VALUE);
+        } else {
+            clientTimezone.setText(TimezoneRegistry.getGMTString(string));
+        }
 
         longOperationsCheck.setSelection(store.getBoolean(DBeaverPreferences.AGENT_LONG_OPERATION_NOTIFY));
         longOperationsTimeout.setSelection(store.getInt(DBeaverPreferences.AGENT_LONG_OPERATION_TIMEOUT));
     }
 
+    @Override
+    public boolean isValid() {
+        return super.isValid() && clientTimezone != null &&
+            (Arrays.stream(clientTimezone.getItems()).anyMatch(s -> s.equals(clientTimezone.getText())));
```

Comment:
```suggestion
        return super.isValid() && ArrayUtils.contains(clientTimezone.getItems(), clientTimezone.getText());
```

How come the `clientTimezone` can be null here?